internal/handlers/common/filter.go

```diff
@@ -603,6 +610,30 @@ func filterFieldExprSize(fieldValue any, sizeValue any) (bool, error) {
 	return true, nil
 }
 
+// filterFieldExprSize handles {field: {$all: [value, another_value, ...]}} filter.
+// The main purpose of $all is to filter arrays.
+// In principle, it is possible to filter non-arrays: {field: {$all: [value]}},
+// but such statement is equivalent to {field: value}.
```

Comment:
I suggest removing it as it doesn't add anything to the function description.

```suggestion
// It is possible to filter non-arrays: {field: {$all: [value]}}, but such statement is equivalent to {field: value}.
```